libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/ApolloCall.kt

```diff
@@ -125,7 +126,26 @@ class ApolloCall<D : Operation.Data> internal constructor(
    * ```
    */
   fun toFlow(): Flow<ApolloResponse<D>> {
-    val request = ApolloRequest.Builder(operation)
+    return apolloClient.executeAsFlow(toApolloRequest(), ignoreApolloClientHttpHeaders = ignoreApolloClientHttpHeaders == true, false)
+  }
+
+  /**
+   * A version of [execute] that restores 3.x behaviour:
+   * - throw on fetch errors.
+   * - make `CacheFirst`, `NetworkFirst` and `CacheAndNetwork` policies ignore fetch errors.
+   * - throw ApolloComposite exception if needed.
+   */
+  @Deprecated("Use toFlow() and handle ApolloResponse.exception instead")
```

Comment:
Should we add the `replaceWith`?

Reply:
I prefer not to because it's not a "drop-in" replacement, the callsite needs to be changed. I don't want users to press Alt-Enter in automatic mode when going through deprecations.

Reply:
Makes sense 👍